storage/2020-08-04/file/files/range_get_file.go

```diff
@@ -8,35 +8,45 @@ import (
 	"runtime"
 	"sync"
 
-	"github.com/Azure/go-autorest/autorest"
+	"github.com/hashicorp/go-azure-sdk/sdk/client"
 )
 
+type GetFileInput struct {
+	Parallelism int
+}
+
+type GetFileResponse struct {
+	HttpResponse *client.Response
+
+	OutputBytes []byte
+}
+
 // GetFile is a helper method to download a file by chunking it automatically
-func (client Client) GetFile(ctx context.Context, accountName, shareName, path, fileName string, parallelism int) (result autorest.Response, outputBytes []byte, err error) {
+func (c Client) GetFile(ctx context.Context, shareName, path, fileName string, input GetFileInput) (resp GetFileResponse, err error) {
 
 	// first look up the file and check out how many bytes it is
-	file, e := client.GetProperties(ctx, accountName, shareName, path, fileName)
+	file, e := c.GetProperties(ctx, shareName, path, fileName)
 	if err != nil {
-		result = file.Response
+		resp.HttpResponse = file.HttpResponse
 		err = e
 		return
 	}
 
 	if file.ContentLength == nil {
-		err = fmt.Errorf("Content-Length was nil!")
+		err = fmt.Errorf("Content-Length was nil")
```

Comment:
we probably also want to set the http response here:

```suggestion
		resp.HttpResponse = file.HttpResponse
		err = fmt.Errorf("Content-Length was nil")
```